pinot-query-planner/src/test/resources/queries/WindowFunctionPlans.json

```diff
@@ -1049,6 +1238,19 @@
           "\n"
         ]
       },
+      {
+        "description": "single OVER(ORDER BY) row_number and select col",
+        "sql": "EXPLAIN PLAN FOR SELECT a.col1, ROW_NUMBER() OVER(ORDER BY a.col2) FROM a",
+        "output": [
```

Comment:
It's hard to review tests with constant scroll up and down to see if the test that comes to my mind has been added or not

So please feel free to ignore / resolve if already added the following 

```
SELECT a.col1, ROW_NUMBER() OVER(ORDER BY a.col2) 
FROM a ORDER BY a.col1
```

```
SELECT a.col1, ROW_NUMBER() OVER(ORDER BY a.col2) 
FROM a ORDER BY a.col2
```

```
SELECT a.col1, ROW_NUMBER() AS rn OVER(ORDER BY a.col2)
 FROM a ORDER BY rn DESC
```

Reply:
done as part of https://github.com/apache/pinot/pull/10684